### DISCUSSION TITLE
Sponsor profile is not verified.

### DISCUSSION THREAD

**Discussion by JavierMartinez07** [community/community#140810] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My sponsor's Github profile is not being verified and says (Your profile is not active yet. We need more information to proceed). Can someone help me a little please? :)

![image](https://github.com/user-attachments/assets/f39e8968-8bbc-44e1-9d19-6f4ba42ce1ef)

**Comment by bfox200925** [comment, marked as answer]
Here are some steps that might help you troubleshoot and move forward:

**Check Email or Notifications:**
 GitHub may have sent an email or in-app notification asking for more information. Be sure to check all associated emails (and spam folders) or your GitHub notifications.

**Submit Missing Information:**
 If there’s a request for specific details, make sure you provide it as soon as possible. Sometimes they need things like verification of your identity, legal name, ect.

**Revisit Profile Setup:**
 Double-check the profile setup and ensure all sections are completed accurately and any other mandatory fields are filled out. Missing or incomplete data can cause delays.

**Respond to GitHub Support:**
 Since you’ve already reached out to support and didn’t get much more than a generic response, consider replying again or submitting another ticket for clarification. It might be worth noting how long you’ve been waiting and asking if there’s any way to expedite the process.

Hope this help! Best of luck to you!
